examples/platform/silabs/ldscripts/SiWx917.ld

```diff
@@ -1,67 +1,54 @@
-/*
+/***************************************************************************//**
```

Comment:
Why we are still having this linker
and the linker is not matching with the slc on dual flash